extensions/shared/plan-upgrade-notification.js

```diff
@@ -7,20 +7,47 @@ import { __ } from '@wordpress/i18n';
 import '@wordpress/notices';
 import { parse as parseUrl } from 'url';
 
+/**
+ * Internal dependencies
+ */
+import getSiteFragment from './get-site-fragment';
+import { isSimpleSite } from './site-type-utils';
+
 /**
  * Shows a notification when a plan is marked as purchased
  * after redirection from WPCOM.
  */
 
+/**
+ * Returns a URL where the current site's plan can be viewed from.
+ * [Relative to current domain for JP sites]
+ *
+ * @return {string} A URL where the current site plan is viewable.
+ */
+function getPlanUrl() {
+	if ( undefined !== typeof window && window.location ) {
+		if ( isSimpleSite() ) {
+			return `https://wordpress.com/plans/my-plan/${ getSiteFragment().replace( '::', '/' ) }`;
```

Comment:
`getSiteFragment()` returns fragment with `::` and that's needed for these wpcom URLs, `replace()` thing can be removed